Library/Formula/vamp-plugin-sdk.rb

```diff
@@ -0,0 +1,24 @@
+
+class VampPluginSdk < Formula
+  homepage "http://www.vamp-plugins.org/"
+  url "http://sourceforge.net/projects/vamp/files/vamp-plugin-sdk/2.2.1/vamp-plugin-sdk-2.2.1.tar.gz/download"
+  version "2.2.1"
+  sha256 "571481098270133d2b78c6a461b850e04a98ab38284227c4d8056385f6333c26"
+
+  depends_on 'pkg-config' => :build
+  depends_on 'libsndfile'
+
+  def patches
```

Comment:
In which way?

You control the order with whatever order you put the ` patch do ` block in. The first patch will be processed and applied first, etc.